apps/beeswax/src/beeswax/common.py

```diff
@@ -44,7 +44,7 @@
 
 RELATION_OPS = ['=', '<>', '<', '<=', '>', '>='] + RELATION_OPS_UNARY
 
-COMPUTE_TYPES = ['hive-compute', 'impala-compute']
+COMPUTE_TYPES = ['hive-compute', 'impala-compute', 'trino-compute', 'trino']
```

Comment:
Why are we adding both `trino-compute` and `trino`?

Reply:
cleaned up. thanks.